judgels-backends/judgels-commons/judgels-service-core/src/main/java/judgels/service/ServiceUtils.java

```diff
@@ -60,6 +65,39 @@ public static Response buildDownloadResponse(String fileUrl) {
         }
     }
 
+    public static Response buildImageResponseFromText(String text, Date lastModifiedStream) {
+        int fontSize = 14;
+        int margin = 30;
+        int charWidth = 8;
+        int charHeight = 17;
+
+        String[] textList = text.split("\\r?\\n");
+        Font font = new Font(Font.MONOSPACED, Font.PLAIN, fontSize);
+        int longestText = Arrays.asList(textList).stream().map(String::length).max(Integer::compareTo).get();
+        int width = charWidth * longestText + 2 * margin;
```

Comment:
```suggestion
        int width = Math.max(700, charWidth * longestText + 2 * margin);
```

Can we do this? So that the width always fully occupies the dialog even when the code is short.